### PR TITLE
Feature sprint06 am 417 create management command

### PR DESCRIPTION
**Pull request by evetion**

### CODE REVIEW

File: delft3dworker/management/commands/containersync_sceneupdate.py [hunk 1]
```diff
@@ -1,5 +1,10 @@
 from django.core.management import BaseCommand
 
+from delft3dcontainermanager.tasks import get_docker_ps
+from delft3dcontainermanager.tasks import do_docker_remove
+from delft3dworker.models import Container
+from delft3dworker.models import Scene
+
 """
 TODO: write this management command with the following functionality:
```
Comment (evetion) [inline delft3dworker/management/commands/containersync_sceneupdate.py]: Remove line

File: delft3dworker/test_management.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        'containersync_sceneupdate.Container._update_state_and_save')
+    @patch('delft3dcontainermanager.tasks.Client', **mock_options)
+    def test_containersync_sceneupdate(self, mockClient, mockContainerupdate):
+        """
+        Test match matrix for docker containers and model containers
+        """
+        client = mockClient.return_value
+        client.containers.return_value = [{'Id': 'abcdefg'},
+                                          {'Id': 'orphan'}]
+        # container = mockContainer.return_value
```
Comment (evetion) [inline delft3dworker/test_management.py]: Remove line

File: delft3dworker/management/commands/containersync_sceneupdate.py [hunk 3]
```diff
@@ -15,4 +19,48 @@ class Command(BaseCommand):
...
+        # retrieve containers from docker
+        containers_docker = get_docker_ps()
+        docker_dict = {x['Id']: x for x in containers_docker}
+        docker_set = set(docker_dict.keys())
+
+        # retrieve container from database
+        container_set = set(
+            Container.objects.values_list('docker_id', flat=True))
+
+        # Work out matching matrix
```
Comment (tschmits) [inline delft3dworker/management/commands/containersync_sceneupdate.py]: a piece of art

File: delft3dworker/management/commands/containersync_sceneupdate.py [hunk 4]
```diff
@@ -15,4 +19,48 @@ class Command(BaseCommand):
     help = "sync containers with container and scene model"
 
     def handle(self, *args, **options):
-        self.stdout.write("containersync_sceneupdate skeleton")
+
+        # Loop over non empty celery_task_ids in containers
+        # celery_set = set(
+            # Container.objects.exclude(celery_id__exact='').values_list('celery_id', flat=True))
```
Comment (tschmits) [inline delft3dworker/management/commands/containersync_sceneupdate.py]: Please do not leave commented code without explanation of why it's commented. So either:
- remove completely
- add explanation why you have this in the file
- activate code by removing hashes

File: delft3dworker/management/commands/containersync_sceneupdate.py [hunk 5]
```diff
@@ -15,4 +19,48 @@ class Command(BaseCommand):
...
+
+        # retrieve containers from docker
+        containers_docker = get_docker_ps()
```
Comment (tschmits) [inline delft3dworker/management/commands/containersync_sceneupdate.py]: Should this not be called like so:

```
get_docker_ps.delay()

try:
    containers_docker = get_docker_ps.get(timeout=30)
except celery.exceptions.TimeoutError as e:
    logging.exception("get_docker_ps timed out (30 seconds)")

if containers_docker is not None:
    ....
```

It seems like calling it directly will not get the remote `docker ps`, but an unintended local one.

File: delft3dworker/management/commands/containersync_sceneupdate.py [hunk 6]
```diff
@@ -15,4 +19,48 @@ class Command(BaseCommand):
...
+        container_set = set(
+            Container.objects.values_list('docker_id', flat=True))
+
+        # Work out matching matrix
+        #       docker  yes non
+        # model x
+        # yes           1_1 1_0
+        # no            0_1 0_0
+        #
+        m_1_1 = docker_set & container_set
```
Comment (tschmits) [inline delft3dworker/management/commands/containersync_sceneupdate.py]: For perfection's sake:

```
m_1_1 =  container_set & docker_set
```

as the first `1` depicts the `container_set` :)

File: delft3dworker/management/commands/containersync_sceneupdate.py [hunk 7]
```diff
@@ -15,4 +19,48 @@ class Command(BaseCommand):
     help = "sync containers with container and scene model"
 
     def handle(self, *args, **options):
```
Comment (tschmits) [inline delft3dworker/management/commands/containersync_sceneupdate.py:23]: This method is a piece of art, truly.
- solid reasoning
- clearly readable
- under 60 lines of code

**Pull request merged**